docs/source/reference/services.md

```diff
@@ -30,15 +29,16 @@ Currently, these characteristics distinguish two types of Services:
 A Service may have the following properties:
```

Comment:
This list doesn't mention `oauth_client_id` or `oauth_redirect_uri` yet

Reply:
Good point, Added.